ticdc/ticdc-faq.md

```diff
@@ -193,10 +193,13 @@ TiCDC uses etcd in PD to store and regularly update the metadata. Because the ti
 
 TiCDC provides partial support for large transactions (more than 5 GB in size). Depending on different scenarios, the following risks might exist:
 
+- The latency of primary-secondary replication might greatly increase.
 - When TiCDC's internal processing capacity is insufficient, the replication task error `ErrBufferReachLimit` might occur.
 - When TiCDC's internal processing capacity is insufficient or the throughput capacity of TiCDC's downstream is insufficient, out of memory (OOM) might occur.
 
-If you encounter an error above, it is recommended to use BR to restore the incremental data of large transactions. The detailed operations are as follows:
+Since v6.2, TiCDC supports splitting a single-table transaction into multiple transactions. This can greatly reduce the latency and memory consumption of replicating large transactions. Therefore, if your application does not have a high requirement on transaction atomicity, it is recommended to enable the splitting of large transctions to solve the possible replication latency and OOM issues. To enable the splitting, set the value of the sink uri parameter [`transaction-atomocity`](/ticdc/manage-ticdc.md#configure-sink-uri-with-mysqltidb) to `none`.
```

Comment:
```suggestion
Since v6.2, TiCDC supports splitting a single-table transaction into multiple transactions. This can greatly reduce the latency and memory consumption of replicating large transactions. Therefore, if your application does not have a high requirement on transaction atomicity, it is recommended to enable the splitting of large transactions to avoid possible replication latency and OOM. To enable the splitting, set the value of the sink uri parameter [`transaction-atomicity`](/ticdc/manage-ticdc.md#configure-sink-uri-with-mysqltidb) to `none`.
```